src/libraries/System.Text.RegularExpressions/gen/Stubs.cs

```diff
@@ -42,6 +42,21 @@ public static string Create<TState>(int length, TState state, SpanAction<char, T
             action(span, state);
             return span.ToString();
         }
+
+        public static int CommonPrefixLength(this ReadOnlySpan<char> span, ReadOnlySpan<char> other)
```

Comment:
have you considered making this logic generic and moving it to a standalone file that could be referenced from other .NET Standard libs like `System.Text.Json` for `Utf8JsonReader.MultiSegment.cs`? Or it's just not worth it for two usages?

Reply:
I don't think it's worth it.  If we find more downlevel, we can reconsider.